src/aws_encryption_sdk/__init__.py

```diff
@@ -36,6 +36,13 @@
     StreamEncryptor,
 )
 
+warnings.warn(
+    'This major version (1.x) of the AWS Encryption SDK for Python has reached End-of-Support.\n' +
+    'It will no longer receive security updates or bug fixes.\n' +
```

Comment:
nit: This isn't flake8 compliant, but given the circumstances, I am fine with ignoring it.